src/code/z_player_lib.c

```diff
@@ -501,12 +570,12 @@ s32 Player_ActionToMagicSpell(Player* this, s32 actionParam) {
     }
 }
 
-s32 Player_HoldsHookshot(Player* this) {
+s32 Player_IsHoldingHookshot(Player* this) {
     return (this->heldItemActionParam == PLAYER_AP_HOOKSHOT) || (this->heldItemActionParam == PLAYER_AP_LONGSHOT);
 }
 
-s32 func_8008F128(Player* this) {
-    return Player_HoldsHookshot(this) && (this->heldActor == NULL);
+s32 Player_IsShootingHookshot(Player* this) {
+    return Player_IsHoldingHookshot(this) && (this->heldActor == NULL);
 }
```

Comment:
links heldActor pointer is set to the hookshot even when hes aiming it or just holding it in his hand. So this name is not accurate.
We will need to distinguish the difference between this and only have the held item action parameter set to the hookshot. 
Again, I recommend leaving unnamed and ill get to it in my items documentation